src/services/contentful/contentful.ts

```diff
@@ -63,40 +78,52 @@ async function getContentfulImage(assetId, _config) {
   }
 }
 
-function parseFeaturedMaps(data, locale = 'en') {
+function parseFeaturedMaps(data: FeaturePlaceItemProps[], locale = 'en') {
   const allItems = data.map((p) => p.fields);
 
   // eslint-disable-next-line no-shadow
-  return data.reduce(async (acc, data) => {
+  return data.reduce(async (acc: any, data) => {
```

Comment:
This any maybe is not needed is the same as not declaring it. Do we know how the acc will look like? Maybe an array of data